### ISSUE TITLE
CI report version mismatch between ruby and gemfile

### ISSUE THREAD

**Issue by jingsam** [dependabot/dependabot-script#453]
```
$ bundle install -j $(nproc) --path vendor
Your Ruby version is 2.6.6, but your Gemfile specified 2.6.5
ERROR: Job failed: exit code 1
```

**Comment by chriske** [comment]
Same here

**Comment by RogierW** [comment]
Can we just bump the Ruby version in the Gemfile to 2.6.6? Or isn't it that simple?

**Comment by awachowiak** [comment]
Same here

**Comment by RogierW** [comment]
It should be solved now; https://github.com/dependabot/dependabot-script/pull/454

**Comment by jingsam** [comment]
@RogierW Does `Gemfile.lock` also need to be regenerate?

**Comment by RogierW** [comment]
@jingsam Maybe it does, I'm not sure actually.

**Comment by awachowiak** [comment]
After changing `Gemfile` file, `Gemfile.lock` should also be updated (regenrated). Both changes should be provided in one commit.

**Comment by RogierW** [comment]
Yes, I see it now. I have made a new pull request; https://github.com/dependabot/dependabot-script/pull/455

**Comment by jingsam** [comment]
What if image `dependabot/dependabot` upgrade ruby to 2.6.7? I guess same issue happens again. I suggest to use versioned container image in `.gitlab-ci.yml`

**Comment by awachowiak** [comment]
Using tagged image in `.gitlab-ci.yml` is good idea.
```
image: dependabot/dependabot-core:0.118.7
```

**Comment by RogierW** [comment]
I agree, that would be a better solution.

**Comment by johannesspohr** [comment]
To anyone still experiencing this make sure to check out `main` and not `master` (speaking from experience)

**Comment by jeffwidman** [comment]
We've updated this repo's README with a [deprecation note](https://github.com/dependabot/dependabot-script?tab=readme-ov-file#this-repo-is-deprecated) which points to a [new example repo](https://github.com/dependabot/example-cli-usage/) we setup to demonstrate how to self-host Dependabot using the [Dependabot CLI](https://github.com/dependabot/cli). As a result, we are archiving this repo.
